src/file.cpp

```diff
@@ -62,6 +62,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #ifdef TORRENT_DISK_STATS
 #include "libtorrent/io.hpp"
+#include "libtorrent/storage.hpp"
```

Comment:
Ok, I will remove it. It's something I'm not worried about, since I know this silly_hash is not expensive.